dev/ci/ci-basic-overlay.sh

```diff
@@ -31,11 +46,43 @@
 ########################################################################
 # Unicoq + Mtac2
 ########################################################################
+
+# Picking:
+#
+# Before picking this was on c33e66c8f2924449c7b98aab108d97b5ee105bab
+# = Nov 4 2019 master
+#   "Merge pull request #28 from validsdp/primitive-floats"
+# - This commit seems to be required for 8.11 since primitive floats are included in 8.11
+#
+# There are no 8.11 specific tags or branches
+# - there are 8.10 tags and branches => request tag
+#
+# The latest 8.10 tag v1.3.2-8.10
+# - has just 5 commit from Jan 2 2020 since Mar 15 2019
+# - does not include a port of the commit used before picking
+#   => looks old
+#
+# The commits on master beyond c33e66c look like adoptions to changes not in 8.11
+#
+# => Continue to use c33e66c8f2924449c7b98aab108d97b5ee105bab from Nov 4 2019
+# => Ask upstream to tag this as 8.11.0
```

Comment:
Or rather, ask upstream to tag this as `v...-8.11`. Reusing version numbers from Coq for external packages is a practice that was started in coq-contribs but that we should not encourage beyond.

Reply:
@Zimmi48 : I am not sure I understand what you mean. With my comment I intended to say that I want to ask upstream to tag the picked version such that it is obvious that this is the version included in 8.11.0. For the sake of speed I don't care that much if this is done before or after the release - for me it doesn't matter much if we use the hash or a tag with the same hash. But for long term maintenance it is helpful to tag it like this, even after the facts, so that people looking at the upstream repo understand the history.

Reply:
I totally support this. I was just saying: don't suggest to project maintainers to literally use "8.11.0" as the tag name. But maybe that was not your intention indeed.